data/plugins/GTAIV.EFLC.FusionFix.ini

```diff
@@ -39,17 +39,17 @@ VolFogFarClip = 4500.0                        // override far clip for volumetri
 ExtendedTimecycEditing = 0                    // 0: disabled | 1: display debug info and reload .dat with F3
 
 [BudgetedIV]
-VehicleBudget = 0                             // may cause issues, set to e.g. 260000000 to increase budget limit
+VehicleBudget = 0                             // used to avoid "Taxi Bug", may cause vehicle audio issue, watch TJGM's video for more information
```

Comment:
```suggestion
VehicleBudget = 0                             // used to avoid "Taxi Bug", may cause vehicle audio issues, watch TJGM's video for more information
```